app/models/nonprofit.rb

```diff
@@ -180,6 +181,10 @@ def as_json(options = {})
     h
   end
 
+  def state_in_us?
```

Comment:
Two things on this line:

* Methods shouldn't end with a question mark in Ruby unless they return a boolean. Validation methods for models don't return a boolean, therefore, you should remove the question mark.

* I'm not sure this is the best name for this method. It's not wrong exactly but if we generalize the set of state codes allowed, we'll have to rename it. Is there a name that makes sense more generally?

Reply:
About the "?",  we just mistyped it. Thanks for the correction!

Regarding the name of the method, what about replacing it to "is_state_valid" or "state_is_valid" so it doesn't look like a question (which would infer a boolean return type)? This way we think it would be more general.

Reply:
I think `state_is_valid` would be good. Let's go with that!